.github/workflows/ios-ci.yml

```
@@ -120,11 +123,11 @@ jobs:
             } |
             select(.functions | length > 0)
           ' "$coverage_json" | \
-          jq -s '
+          jq -s --arg pwd "$current_dir" '
             group_by(.path) | 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)